spec/app/domain/authentication/authenticate_spec.rb

```diff
@@ -94,8 +75,11 @@ def authenticator(pass:)
       )
 
       Authentication::Authenticate.new(
-        token_factory: token_factory,
-        audit_log: audit_logger
+        validate_role_can_access_webservice: mock_validate_user_can_access_webservice(validation_succeeded: true),
```

Comment:
Because you changed `validate_user_can_access_webservice` -> `validate_role_can_access_webservice`, please follow suit here (: